ArduPlane/mode_initializing.cpp

```diff
@@ -0,0 +1,12 @@
+#include "mode.h"
+#include "Plane.h"
+
+bool ModeInitializing::_enter()
+{
+    plane.throttle_allows_nudging = true;
+    plane.auto_throttle_mode = true;
```

Comment:
What? We allow automatic throttle control while booting? Nudging maybe, but throttle seems off.

Reply:
Hmm.. good catch. Maybe this has been always been the cause of those random throttle-up-on-boot bugs we hear about occasionally?